docset/winserver2012-ps/activedirectory/Get-ADUser.md

```diff
@@ -118,6 +118,20 @@ Description
 
 Get the user with name 'GlenJohn' on the AD LDS instance.
 
+### -------------------------- EXAMPLE 5 --------------------------
+```powershell
+C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
+```
+
+Description
+
+-----------
```

Comment:
```suggestion
```
Thank you @doctordns for this contribution!

Let's remove these lines in the examples.